lib/pushmi_pullyu/logging.rb

```diff
@@ -23,8 +23,8 @@ def self.logger
     @logger ||= initialize_logger
   end
 
-  def self.log_aip_activity(noid, message)
-    log_file = "#{PushmiPullyu.options[:workdir]}/#{noid}/data/logs/aipcreation.log"
+  def self.log_aip_activity(aip_directory, message)
+    log_file = "#{aip_directory}/data/logs/aipcreation.log"
```

Comment:
just pass in the aip_directory (which is well known and already calculated when calling this) instead of computing the aip_directory once again here.